site/operational-guide/uniform-fixed-cluster.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Uniform Fixed Clusters
+menu_order: 30
+---
+> Author's Note: I'm going to change this section to recommend systemd
+> at which point the mentions of `weave launch` and `weave connect`
+> will be replaced and I will talk about amending the PEER list in
+> /etc/sysconfig/weave instead
+
+A uniform fixed cluster has the following characteristics:
+
+* Each node has identical configuration
+* There is a controlled process for adding or removing nodes. The end
+  user is responsible for ensuring that only one instance of the
+  process is in-flight at a time
+
+### Bootstrapping
+
+This scenario describes a production deployment of a fixed number of
+N nodes (N=1 in the degenerate case).
+
+On each peer:
+
+    weave launch <initial peer list>
+
+The initial peer list may contain the address of the peer to which it
+is being supplied, so the configuration may be identical for each
+peer.
+
+Then on _any_ peer:
```

Comment:
We may want to get across that it's ok to run it on more than one, even all peers.